### PR TITLE
Chrome console allows accessing private properties

### LINKED ISSUE #25411: Chrome Developer Tools is able to Accesss Private Methods and properties
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Classes_in_JavaScript

### What specific section or headline is this issue about?

Encapsulation

### What information was incorrect, unhelpful, or incomplete?

attempts to accesss private data properties and methods outside a class is expected to produce a syntax error. However, this is not the case when the examples were tried on Chrome Console.

### What did you expect to see?

```myExample.#somePrivateMethod()``` 
Expected a syntax error but got the method's output instead.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/javascript/objects/classes_in_javascript`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Classes_in_JavaScript
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/objects/classes_in_javascript/index.md
* Last commit: https://github.com/mdn/content/commit/4def230f85756724b59660e3cd9de363db724ef8
* Document last modified: 2023-02-24T00:05:28.000Z

</details>

### PR DESCRIPTION
**Pull request by Josh-Cena**
### Description

Fix #25411

### CODE REVIEW

File: files/en-us/web/javascript/guide/using_classes/index.md [hunk 1]
```diff
@@ -352,6 +352,8 @@ Accessing private fields outside the class is an early syntax error. The languag
 console.log(red.#values); // SyntaxError: Private field '#values' must be declared in an enclosing class
 ```
 
+> **Note:** The Chrome console allows accessing private properties outside the class. However, this is a DevTools-only ability and doing the same elsewhere is always a syntax error.
```
Comment (hamishwillee) [inline files/en-us/web/javascript/guide/using_classes/index.md:355]: A few lines above this says "Accessing private fields outside the class is an early syntax error. What does it mean by "early"?

What you have is fine; here is an alternative "for your consideration"

```suggestion
> **Note:** Code run in the Chrome console can access private properties outside the class.
> This is a DevTools-only exception to the JavaScript language requirement that such access should throw a `SyntaxError`.
```

**Review by hamishwillee — APPROVED** [review]

**Comment by Josh-Cena** [comment]
> What does it mean by "early"?

An "early error" is an error thrown at parsing time that prevents the code from being evaluated at all—it's basically "syntax error" except the error message is probably more friendly as the parser is not in an unacceptable state. A "runtime error" is thrown _during_ code evaluation.

File: files/en-us/web/javascript/guide/using_classes/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/guide/using_classes/index.md:355]:
```suggestion
> **Note:** Code run in the Chrome console can access private properties outside the class. This is a DevTools-only extension of the JavaScript language syntax.
```

File: files/en-us/learn/javascript/objects/classes_in_javascript/index.md [hunk 2]
```diff
@@ -178,6 +178,8 @@ summers.canStudyArchery(); // true
 summers.#year; // SyntaxError
 ```
 
+> **Note:** The Chrome console allows accessing private properties outside the class. However, this is a DevTools-only ability and doing the same elsewhere is always a syntax error.
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/objects/classes_in_javascript/index.md:181]: > **Note:** Code run in the Chrome console can access private properties outside the class. This is a DevTools-only relaxation of the JavaScript syntax restriction.
```

File: files/en-us/web/javascript/reference/classes/private_class_fields/index.md [hunk 3]
```diff
@@ -72,6 +72,8 @@ instance.#privateField === 42; // Syntax error
 
 JavaScript, being a dynamic language, is able to perform this compile-time check because of the special hash identifier syntax, making it different from normal properties on the syntax level.
 
+> **Note:** The Chrome console allows accessing private properties outside the class. However, this is a DevTools-only ability and doing the same elsewhere is always a syntax error.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/classes/private_class_fields/index.md:75]:
```suggestion
> **Note:** Code run in the Chrome console can access private properties outside the class. This is a DevTools-only relaxation of the JavaScript syntax restriction.
```

**Comment by Josh-Cena** [comment]
@hamishwillee Thanks for the review! Much better sentence structure than mine. I just polished the words a bit.

**Comment by hamishwillee** [comment]
You're welcome. All good!

**Pull request merged**